pkg/plugins/golang/v3/init.go

```diff
@@ -188,7 +187,7 @@ func checkDir() error {
 				return err
 			}
 			if info.Name() != "go.mod" && !strings.HasPrefix(info.Name(), ".") {
-				return errors.New("only the go.mod and files with the prefix \"(.)\" are allowed before the init")
+				return fmt.Errorf("target directory is not empty: found existing file %q", path)
```

Comment:
NIT: I think the file name should be enough
```suggestion
				return fmt.Errorf("target directory is not empty: found existing file %q", info.Name())
```

Reply:
I started with that actually and switched to path :-) The root cause here was that we were finding files like `.git/HEAD`.  info.Name() prints "HEAD", but path prints ".git/HEAD".

Reply:
Should we not still clarifying what files are allowed? See that any file with the prefix (.) is allowed. E.g. .gitignore

Reply:
My thought was that maybe the user doesn't care, but it is a good idea to try to reference the idea that some files are allowed, so I added it back!  Thanks!